sklearn/cluster/spectral.py

```diff
@@ -362,6 +367,12 @@ class SpectralClustering(BaseEstimator, ClusterMixin):
         The number of parallel jobs to run.
         If ``-1``, then the number of jobs is set to the number of CPU cores.
 
+    copy : bool, optional, default: True
+        If copy is False, then the laplacian matrix is copied in the affinity
```

Comment:
I had meant the laplacian matrix would use the same memory pointed by the affinity matrix. Now I think this is indeed a bit misleading. I will change "copied" to "constructed" as you had pointed out in docstrings in the other files.